src/strategies/anti-whale/README.md

```diff
@@ -0,0 +1,85 @@
+# Anti-Whale strategy
+
+## Description
+
+This strategy executes a configured strategy and applies an anti-whale measure to its results to reduce the impact of big wallets in the the resulting value, reducing the effect on the voting power as the token amount increases.
+
+It will apply the the following to the result:
+  
+  ```none
+  If result > antiWhale.threshold
+    result = antiWhale.inflectionPoint * ( result / antiWhale.inflectionPoint ) ^ antiWhale.exponent
+
+  If result <= antiWhale.threshold {
+    thresholdMultiplier = ( antiWhale.inflectionPoint * ( antiWhale.threshold / antiWhale.inflectionPoint )^antiWhale.exponent ) / antiWhale.threshold
+
+    result = result * thresholdMultiplier
+  }
+  ```
+
+## Accepted options
+
+- **strategy.name:** name of the strategy to run
+
+- **strategy.network:** the network id to be used by the strategy.
```

Comment:
The issue with that is that the space has already a network defined, and the snapshot block number is specific to this network, if you use Polygon network on the strategy but the space use mainnet the snapshot will be completely off. I'd rather not add any network param here.

Reply:
ahh I see.... good one, I guess that's why the multichain strategy made some calls to the Graph to get the blocks for each network :)

Reply:
Exactly!

Reply:
It's done. Thanks for the help!